### DISCUSSION TITLE
[功能建议] 请问是否可以增加两个评论区表情相关的组件

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#4072] [功能建议 / Ideas]
_使用 `颜文字` `黄豆` `表情` 三个关键字搜索，没有看到相关讨论_

期望 `评论区表情替换屏蔽` 组件可以做到：
1.将评论区的表情图片**替换**成其他文字（甚至图片），也提供针对某个表情**定向屏蔽**或**全部屏蔽**的选项
2.重写评论发送栏中的**表情选项**，使用户可以选择**是否显示**选项中提供的每个表情分类，且能够**自由排序**它们

_疯狂的想法！_ 期望 `评论区自定义表情` 组件可以做到：
1.通过编辑组件设置，将评论区中出现的**关键字**与**图片链接**绑定，实现只能在用户客户端使用的独特的**自定义表情**，并选择性地添加到表情选项中
2.通过检索评论区关键字，**重现**来自拥有重复组件设置的用户所发布评论之中的表情

以上，感谢！辛苦了！

**Comment by the1812** [comment]
综合在 `评论内容替换` 组件里实现, 不过目前还不支持 "重写评论发送栏中的表情选项"
